src/lcf/rooms.395

```diff
@@ -7,7 +6,7 @@
 
 <GDECL (MUDDLE) FIX (TENEX?) <OR ATOM FALSE> (VERS DEV SNM SCRATCH-STR) STRING>
 
-<DEFINE SAVE-IT ("OPTIONAL" (FN <COND (<L? ,MUDDLE 100> "CFS;MADADV SAVE")
+<DEFINE SAVE-IT ("OPTIONAL" (FN <COND (<L? ,MUDDLE 100> "LCF;MADADV SAVE")
```

Comment:
Why did you change this?  We discussed having sources in LCF and the save file in CFS.

Reply:
Because the SAVE file should end up in MADMAN, not CFS.

Reply:
The implementers used the script to produce the SAVE file, which was then tested and moved (manually) to the MADMAN directory. It seemed more logical to me to put the NBINs in the LCF directory (the RECORD files will end up there anyway — even if you tell the compiler to put the NBIN in a separate directory).

Reply:
I checked backup records, and I see MADADV SAVE resided in the MADMAN directory until February 1979.  After that it was moved to CFS.  There was also a CFS; MADADV XXFILE, so your hunch was right!  The DOC, HELP, and INFO files were also moved to CFS.